src/Softeq.NetKit.Chat.SignalR/Sockets/ChannelSocketService.cs

```diff
@@ -212,7 +208,7 @@ public async Task DeleteMemberFromChannelAsync(DeleteMemberRequest request)
             await _pushNotificationService.UnsubscribeUserFromTagAsync(memberToDelete.SaasUserId, PushNotificationsTagTemplates.GetChatChannelTag(request.ChannelId.ToString()));
             await _channelNotificationService.OnDeletedFromChannel(memberToDelete, request.ChannelId);
```

Comment:
We also need to modify this method because it's contain YouAreDeleted and MemberDeleted event. The second one is not necessary due the system notification you updated.

Reply:
MemberDeleted  is not using at all